src/test/librados/tier_cxx.cc

```diff
@@ -3621,7 +3658,9 @@ TEST_F(LibRadosTwoPoolsPP, ManifestSnapRefcount) {
     } catch (buffer::error& err) {
       ASSERT_TRUE(0);
     }
-    ASSERT_EQ(1u, refs.count());
+    if (refs.count() != 1u) {
```

Comment:
Why these if checks?  Is refs supposed to have just 1?

Reply:
Yeah, The expected ref is 1 here. If the ref is not 1, just check whether the reference of both source and target is correct or not by the following code.

Reply:
I'm really not following.  Why not just embed that check in is_intended_refcount_state?